spec/models/restaurant_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
```

Comment:
[Run annotate](https://github.com/ctran/annotate_models#usage-in-rails) right now, it should generate the model schema here